src/backend/distributed/commands/multi_copy.c

```diff
@@ -3464,6 +3501,15 @@ CopyGetPlacementConnection(ShardPlacement *placement, bool stopOnFailure)
 		return connection;
 	}
 
+
+	connection =
```

Comment:
one weird thing to note here is: We're getting connections which were previously established  with `CONNECTION_PER_PLACEMENT`.

In our current connection management implantation, that flag is only used during connection establishment in `StartPlacementListConnection`. But, would it become problem if we rely on that flag in the future?

I think we can consider using `FORCE_NEW_CONNECTION` instead of `CONNECTION_PER_PLACEMENT` in COPY?

Reply:
I intentionally did not try to change the behavior here as I do not have a strong opinion to change or keep the current flags.

If a reviewer thinks I should reconsider, please feel free to comment here.